.github/scripts/update-greptimedb-version.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+TAG="v0.7.1"
+
+function replace_tag() {
+  if [ -n "$IMAGE_TAG" ]; then
+      TAG=$IMAGE_TAG
+  fi
+}
+
+function update_greptimedb_standalone_chart() {
+  file_path="charts/greptimedb-standalone/Chart.yaml"
+
+  current_version=$(yq eval '.version' "$file_path")
+
+  major=$(echo "$current_version" | awk -F. '{print $1}')
+  minor=$(echo "$current_version" | awk -F. '{print $2}')
+  patch=$(echo "$current_version" | awk -F. '{print $3}')
+
+  next_version="$major.$minor.$((patch + 1))"
+
+  appVersion="${TAG#v}"
+  yq eval ".appVersion = \"$appVersion\"" -i "$file_path"
+  yq eval ".version = \"$next_version\"" -i "$file_path"
+
+  echo "Chart version updated to $next_version successfully."
+}
+
+function update_greptimedb_standalone_image() {
+  yq e ".image.tag = \"$TAG\"" charts/greptimedb-standalone/values.yaml > /tmp/greptimedb-standalone-values-updated.yaml
```

Comment:
question: why not use `yq eval` to modify the file in-place?

Reply:
The `yq` tool will format the chart line is not expected. more describe: https://github.com/mikefarah/yq/issues/515